src/components/messages/messages-container-header/MessagesContainerHeader.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="messages-container-header">
```

Comment:
If this is a header, shouldn't we use the `header` html tag to be semantically correct?